editor/js/Sidebar.Material.js

```diff
@@ -893,6 +893,8 @@ Sidebar.Material = function ( editor ) {
 
 			}
 
+			signals.materialChanged.dispatch( material );
```

Comment:
`SetMaterialValueCommand` already dispatches `materialChanged`. So this would make it dispatch twice.